core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceAdminService.scala

```diff
@@ -6,28 +6,41 @@ import org.broadinstitute.dsde.rawls.NoSuchWorkspaceException
 import org.broadinstitute.dsde.rawls.dataaccess._
 import org.broadinstitute.dsde.rawls.dataaccess.slick._
 import org.broadinstitute.dsde.rawls.metrics.RawlsInstrumented
-import org.broadinstitute.dsde.rawls.model.{AttributeName, AttributeValue,WorkspaceAttributeSpecs, ErrorReportSource, ManagedGroupRef, RawlsGroupName, RawlsRequestContext, SamResourceTypeName, SamResourceTypeNames, Workspace, WorkspaceDetails, WorkspaceFeatureFlag, WorkspaceName}
+import org.broadinstitute.dsde.rawls.model.{
```

Comment:
These scala files were failing the `scalafmt` check after I pulled in the latest changes, so I ran `scalafmt`, but otherwise have not touched them.  Not sure how they ended up on main in the incorrect format!